internal/core/models/integration_models.go

```diff
@@ -50,4 +51,6 @@ type GetIntegrationQueryResult struct {
 	RefreshLimitSecs             int                    `json:"refresh_limit_secs"`
 	AutoPiPowertrainToTemplateID map[PowertrainType]int `json:"auto_pi_power_train_to_template_id,omitempty"`
 	TokenID                      int                    `json:"token_id"`
+	Points                       int                    `json:"points"`
+	ManufacturerTokenID          *big.Int               `json:"manufacturer_token_id"`
```

Comment:
But it's weird that we use `*big.Int` for this and `int` for `TokenID`.

Reply:
I'd be fine changing it for consistency. But part of me also wonders if we just use uint64 and save the hassle of big.Int.